src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,22 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
```

Comment:
add to String to all models as you will not get proper message displayed in exceptions

Reply:
done.